api/stats.js

```diff
@@ -116,18 +126,22 @@ export class Stats {
             }
         }
 
+        logger.debug("done getting daily stats from channels");
         return stats;
     };
 
     _sendStatsChannel = async (serverStats, title) => {
+        logger.debug(`sending '${title}' stats to channel ${config.STATS_CHANNEL}`);
         if (!serverStats || !Object.keys(serverStats).length) {
             // nothing to update
             return;
         }
-        let inactive = [];
-        let active = [];
-        for (let user in serverStats) {
+        const inactive = [];
+        const active = [];
+        // eslint-disable-next-line guard-for-in
```

Comment:
is this really useful? can we just disable that check completely?

Reply:
True. This one shouldn't have happened. Once we have CI we won't be able to push rules that violate the linter rules (that one got pushed bypassing it)